<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogIDE</class>
 <widget class="QDialog" name="DialogIDE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Page</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Source</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPlainTextEdit" name="editor"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Precomp</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPlainTextEdit" name="outputpp"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>ASM</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPlainTextEdit" name="outputasm"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Output</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPlainTextEdit" name="outputstd"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>LM</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPlainTextEdit" name="outputlm"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savePB">
         <property name="text">
          <string>Save...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Install To</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="injectCB"/>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>At (0x0000)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="origEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="installPB">
              <property name="text">
               <string>Install</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Compile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="modelCB">
         <item>
          <property name="text">
           <string>PC-1350</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogIDE</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogIDE</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
